src/main/java/careconnect/logic/LogicManager.java

```diff
@@ -50,6 +50,8 @@ public class LogicManager implements Logic {
             "Could not save data to file %s due to insufficient permissions to write to the file "
                     + "or the folder.";
 
+    private static Command commandToConfirm = null;
+
```

Comment:
Yep, looks good, logic manager seems to be the most appropriate component to place this responsibility! thanks!